src/components/ReportActionItem/MoneyRequestView.tsx

```diff
@@ -255,6 +275,48 @@ function MoneyRequestView({
         [transactionAmount, isSettled, isCancelled, isPolicyExpenseChat, isEmptyMerchant, transactionDate, hasErrors, canUseViolations, hasViolations, translate, getViolationsForField],
     );
 
+    const distanceRequestFields = canUseP2PDistanceRequests ? (
+        <>
+            <OfflineWithFeedback pendingAction={getPendingFieldAction('waypoints')}>
+                <MenuItemWithTopDescription
+                    description={translate('common.distance')}
+                    title={distanceToDisplay}
+                    interactive={canEditDistance}
+                    shouldShowRightIcon={canEditDistance}
+                    titleStyle={styles.flex1}
+                    onPress={() =>
+                        Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_DISTANCE.getRoute(CONST.IOU.ACTION.EDIT, CONST.IOU.TYPE.REQUEST, transaction?.transactionID ?? '', report.reportID))
+                    }
+                />
+            </OfflineWithFeedback>
+            <OfflineWithFeedback pendingAction={getPendingFieldAction('waypoints')}>
```

Comment:
NAB: We haven't implemented it yet, but eventually this should be `distanceRate` or something like that.

Reply:
@neil-marcellini I'll add a comment, but wouldn't it be better to handle it in another PR?

Reply:
Yep that sounds good. I think we should do it in the PR for the issue where we implement updating the rate on the frontend [[$500] [HOLD 36985] [P2P Distance] Create UpdateMoneyRequestRate in App](https://github.com/Expensify/App/issues/36987)